src/harness/sas_testcase.py

```diff
@@ -11,22 +11,17 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
-"""Implementation of SasTestcaseInterface"""
 
-import json
-from jsonschema import validate, Draft4Validator, RefResolver
 from datetime import datetime
-import os
-import logging
-import unittest
-import sas_interface
-import sas
 import signal
 import time
+import unittest
+import sas
+from google3.third_party.winnforum_sas.src.harness import util
```

Comment:
Should this be the util in the test harness codes ?

Reply:
Fixed. Thanks for catching this!